set.mm

```diff
@@ -127910,12 +127908,12 @@ least element (schema form).  (Contributed by NM, 17-Aug-2001.) $)
        member of the set.  (Contributed by Mario Carneiro, 21-Apr-2015.) $)
     suprzub $p |- ( ( A C_ ZZ /\ E. x e. ZZ A. y e. A y <_ x /\ B e. A ) ->
```

Comment:
The comment contains a statement about axioms. Is this still true?

Reply:
Looking at the axiom usage file https://github.com/GinoGiotto/set.mm/blob/axiom-usage-min22/axiom-usage.log generated from the scan of minimizations up to min22 it seems that it is still true:
<pre>Statement "suprzcl2" assumes the following axioms ($a statements):
  ax-mp ax-1 ax-2 ax-3 ax-gen ax-4 ax-5 ax-6 ax-7 ax-8 ax-9 ax-10 ax-11 ax-12
  ax-13 ax-ext ax-sep ax-nul ax-pow ax-pr ax-un ax-cnex ax-resscn ax-1cn ax-icn
  ax-addcl ax-addrcl ax-mulcl ax-mulrcl ax-mulcom ax-addass ax-mulass ax-distr
  ax-i2m1 ax-1ne0 ax-1rid ax-rnegex ax-rrecex ax-cnre ax-pre-lttri ax-pre-lttrn
  ax-pre-ltadd ax-pre-mulgt0</pre>